src/Sulu/Bundle/CoreBundle/DataFixtures/replacers.xml

```diff
@@ -1211,4 +1211,10 @@
         <column name="from"><![CDATA[&]]></column>
         <column name="to">y</column>
     </item>
```

Comment:
Shouldn't we remove all the replacers just for the letters (so basically everything except the replacers for `&`)? Isn't that handled by the `String` component now? Might lead to confusing situations otherwise.

Reply:
I left them as-is because of BC. If somebody parses and uses them manually.

Reply:
Okay, sounds good :+1: